src/models/index.js

```diff
@@ -63,9 +64,9 @@ if (process.env.NODE_ENV !== "test") {
 	});
 
 	module.exports.artist = connection.define("artist", {
-			id: 1,
-			name: "Micro Jackson"
-		}
+		id: 1,
+		name: "Micro Jackson"
+	}
```

Comment:
Technically not your issue, but the `{` opening this should be on a new line, or the `);` should be on the same line as the `}`. I'm in favour of the latter, considering the rest of the project.